### PR TITLE
adds big csters de-stressing event with TBA place and time

### PR DESCRIPTION
**Pull request by claricen**

### CODE REVIEW

**Comment by evykassirer** [comment]
@techiechelle I'll let you take a look first 😄

File: content/S2016/S16-event-csters-destress.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+As always, food will be provided! (With vegetarian, vegan, gluten-free options.)
+
+Note: This event is [women-only](http://wics.uwaterloo.ca/faq).
+
+## Event Details ##
+
++ **Who:** [Women and non-binary individuals](http://wics.uwaterloo.ca/faq).
++ **What:** WiCS & FemPhys De-stressing Session
++ **Where:** TBA
```
Comment (ehashman) [inline content/S2016/S16-event-csters-destress.md]: Can we get a room on this before we publish?

Comment (evykassirer) [inline content/S2016/S16-event-csters-destress.md]: Depends how soon it's booked, but ideally yes. I'd like to have it posted by the end of the week either way

**Comment by techiechelle** [comment]
QNC 1506

The FemPhys proposed to ask attendees to bring a small container for the face mask? They are going to buy ziploc just in case, though.

File: content/S2016/S16-event-csters-destress.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
++ **When:** Tuesday. June 7, 6:00 PM
```
Comment (evykassirer) [inline content/S2016/S16-event-csters-destress.md]: we now know it'll go until 7:30

@techiechelle what kind of details can we add? Face masks, discussion, anything else?

File: content/S2016/S16-event-csters-destress.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Tags: social, big csters
+Slug: big-csters-destressing-s16
+Author: Clare Ng
+Summary: Join us for a relaxing night of socializing and de-stressing.
+
+WiCS is collaborating with FemPhys to bring you a de-stressing session! We 
+know school can be pretty stressful sometimes, so hopefully you can take 
+an opportunity to take a breather and de-stress with us for a couple hours.
+
+There will be a variety of activities including face-mask making, colouring,
```
Comment (ehashman) [inline content/S2016/S16-event-csters-destress.md]: s/face-mask making/face mask-making/

File: content/S2016/S16-event-csters-destress.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+title: Big CSters: WiCS & FemPhys De-stressing Session
+Date: 2016-6-1
+Category: Events
+Tags: social, big csters
+Slug: big-csters-destressing-s16
+Author: Clare Ng
+Summary: Join us for a relaxing night of socializing and de-stressing.
+
+WiCS is collaborating with FemPhys to bring you a de-stressing session! We
```
Comment (fboxwala) [inline content/S2016/S16-event-csters-destress.md]: Do we want to link to femphys' facebook page?
https://www.facebook.com/groups/705741329511034/?fref=nf

File: content/S2016/S16-event-csters-destress.md (hunk 3 above)
Comment (evykassirer) [inline content/S2016/S16-event-csters-destress.md]: "making face masks"?

**Pull request merged**